### ISSUE TITLE
Simple Celsius to Fahrenheit Model Fails

### ISSUE THREAD

**Issue by DennisFaucher** [uber/ludwig#270]
**Describe the bug**
Not sure what I am doing wrong. Created the simple task of learning the C to F calculation. Model predicted that 100 C is -27 F (Should be 212 F). Thank you.

**To Reproduce**
See this Google Colab: https://colab.research.google.com/drive/1kg6LlklDLdXHAqVUohJILRj5cJC0FznX

Please provide code, yaml definition file and a sample of data in order to entirely reproduce the issue.
Issues that are not reproducible will be ignored.

**Expected behavior**
100 C should be predicted as 212 F

**Screenshots**
![image](https://user-images.githubusercontent.com/9034190/55519293-34cb9480-5645-11e9-96e2-f398f3bb5a23.png)
![image](https://user-images.githubusercontent.com/9034190/55519336-604e7f00-5645-11e9-9ef9-91f70111b5b7.png)
![image](https://user-images.githubusercontent.com/9034190/55519346-68a6ba00-5645-11e9-9f12-0722872ff90b.png)

**Environment (please complete the following information):**
 - OS: [e.g. iOS] Google Colab Linux
- Python version: 3
- Ludwig version Latest

**Additional context**

**Comment by w4nderlust** [comment]
The colab looks wrong, looks like an image classification. Please provide the correct zip containing dataset csv and yaml file so that we can debug locally please.

**Comment by DennisFaucher** [comment]
My apologies.
Correct link: https://colab.research.google.com/drive/1occqmEwSuGkJWao5B4uHouaD3xEwx08l
Files attached
[c2f.zip](https://github.com/uber/ludwig/files/3044249/c2f.zip)

**Comment by w4nderlust** [comment]
With this model it gets mean absolute error of 0.2314:
```
input_features:
    -
        name: C
        type: numerical
        initializer: ones
output_features:
    -
        name: F
        type: numerical
        initializer: ones
training:
  learning_rate: 0.00005
  optimizer:
    type: sgd
```
There are two issues here: the absolute value of inputs is big, so you need something that initializes to small values (hence ones) to avoid explosion. At the same time because inputs and outputs are not normalzied, you need a small enough learning rate to avoid divergence because of big gradients.

**Comment by DennisFaucher** [comment]
@w4nderlust thank you very much. I'll read up on the statements you added to the YAML.
